### ISSUE TITLE
minor: Not selecting any layers -> invalid javascript

### ISSUE THREAD

**Issue by carolinux** [Geolicious/qgis2leaf#211]
My first use of the plugin and I forgot to add any layers -> the result was some javascript which was missing a '{'. I think the plugin could warn when trying to do this. 

When I actually added a layer, no complaints :)

**Comment by tomchadwin** [comment]
No, it's important because some people might need simply an embedded OSM. Will investigate.

**Comment by carolinux** [comment]
I found the problem. Will make a PR soon of a possible fix.

**Comment by tomchadwin** [comment]
So is it incorrect even if some layers are selected and included in the map? We don't want to introduce errors in other circumstances.

**Comment by carolinux** [comment]
Yes, apparently this is so to delete the last comma added when layers are added.

**Comment by carolinux** [comment]
You can use
````
",".join("foo","bar","baz")
```
renders foo,bar,baz 
no need to worry about the last comma
